bhohnco/Rancid-Tomatillos
# Feature single movie card build

## Is this a fix or a feature?
Feature

## What is the change?
error messages are now showing for 500 errors/API errors 

## What does it fix?
The user can now see when there's a problem with the API call 

## Where should the reviewer start?
The app.js file and the API file 

## How should this be tested?
remove a character from the API HTML address in the API fetch file, and then try to load the page to see the error generate.
